libs/react-helpers/src/lib/local-logger.ts

```diff
@@ -0,0 +1,119 @@
+import * as Sentry from '@sentry/browser';
+import type { SeverityLevel, Scope } from '@sentry/browser';
+import type { Severity, Breadcrumb, Primitive } from '@sentry/types';
+
+type ConsoleArg = string | number | boolean | bigint | symbol | object;
```

Comment:
suggestion (non blocking): 
```
type ConsoleArg = Parameters<typeof console.log>
```

Reply:
It would break a little bit further concept of typing params to the sentry :-(